platforms/NintendoWii.cmake

```diff
@@ -0,0 +1,55 @@
+find_package(PkgConfig REQUIRED)
+
+add_executable(RetroEngine ${RETRO_FILES})
+
+set(RETRO_SUBSYSTEM "GX" CACHE STRING "The subsystem to use")
+if(NOT (RETRO_SUBSYSTEM STREQUAL "GX"))
+    message(FATAL_ERROR "RETRO_SUBSYSTEM must be set to GX")
+endif()
+if(NOT GAME_STATIC) 
+    message(FATAL_ERROR "GAME_STATIC must be on")
+endif()
+
+set(RETRO_MOD_LOADER OFF CACHE BOOL "Disable the mod loader" FORCE)

```

Comment:
`RETRO_MOD_LOADER` should be ON by default now (also the description is inverted, it should *Enable* the mod loader).

Reply:
Is mod loader working on 3DS? The Makefile configuration still lists it as off.

Reply:
On 3DS it's still disabled, but I've enabled it on the Wii so that sound support for Legacy modes is easier to enable.

Reply:
why did i type 3ds i meant wii

Reply:
it's also not in the makefile im just makin shit up left and right